src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    private Color color;
+    private double area;
```

Comment:
let's remove this parameter as we obtain area dynamically through getArea method